### PR TITLE
Feature: Add Test For Item Counter

### PR DESCRIPTION
**Pull request by Whoistolu**
In this milestone, I:
- Made sure that function for the items counter is covered with unit tests using Jest.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -51,14 +55,12 @@ const createCard = (imagePara, numberOfLikes) => {
...
 
-const itemCalculator = (itemArray) => itemArray.length;
-
 const fetchAllPokemonNames = async () => {
   const result = await fetch(`${baseURL}pokemon?limit=9`);
   const data = await result.json();
   const allLikes = await apiLikes();
   const spaceship = document.querySelector('.spaceship');
-  spaceship.textContent = `Spacehips (${itemCalculator(data.results)})`;
+  spaceship.textContent = `POKEMAN (${itemCalculator(data.results)})`;
```
Comment (Lordkaito) [inline src/index.js]: Kindly change the text content here from "POKEMAN" to "POKEMON"

File: dist/index.html [hunk 2]
```diff
@@ -18,10 +18,10 @@
       <p class="spaceship"></p>
     </div>
     <div class="planet item">
-      <p>Planets</p>
+      <a href="#">ABOUT</a>
     </div>
     <div class="race item">
-      <p>Races</p>
+      <a href="#">CONTACT</a>
```
Comment (Lordkaito) [inline dist/index.html:24]: Kindly change your classes on these elements to something related to what you have inside the tags, so the code will be more semantic.

**Review by Lordkaito — APPROVED** [review]
# STATUS: APROVED ✔️ 
### Good job
Just a few changes that would make the code better, consider them optional but it would be much better if you apply those changes.
Check at the inline comments to find those changes.
### No more things to say, good job

File: src/index.js (hunk 1 above)
Comment (Whoistolu) [reply src/index.js]: ## Hello @Lordkaito ,
### Thank you for your feedback.
Changes will be implemented immediately.

File: dist/index.html (hunk 2 above)
Comment (Whoistolu) [reply dist/index.html:24]: ## Hello @Lordkaito ,
### Thank you for your feedback.
- Changes will be implemented immediately.

**Pull request merged**